src/config.ts

```diff
@@ -54,14 +54,16 @@ export const SCENE_TILE_PROVIDERS = [
     maxZoom:  13,
     name: 'Copernicus Sentinel-2',
     prefix: 'sentinel',
+    catalogSection: 'planet', // Look in /planet/* in the catalog
     provider: 'Planet',
     url: `https://tiles{0-3}.${PLANET_BASE_URL}/v1/experimental/tiles/Sentinel2L1C/__SCENE_ID__/{z}/{x}/{y}.png?api_key=__API_KEY__`,
   },
   {
     attributions: '&copy; <a href="https://www.planet.com" target="_blank" rel="noopener">Planet Labs (Landsat8)</a>',
     maxZoom:  13,
-    name: 'Landsat8',
+    name: 'Landsat8 (Planet)',
```

Comment:
We populate that dropdown by saying "name (provider)" so this would make it be Landsat 8 (Planet) (Planet)

Reply:
Fixed